routes/route.js

```diff
@@ -0,0 +1,58 @@
+const categorieDB = require("../model/categories");
+const express = require("express");
+const itemDB = require("../model/items");
+const router = express.Router();
+
+// rendering homepage
+router.get("/", (req, res) => {
+  res.render("homepage");
+});
+
+//rendering categories
+router.get("/categories", async (req, res) => {
+  const categorieData = await categorieDB.find();
+  res.render("categories", { cat: categorieData });
+});
+
+//rendering items
+router.get("/items", async (req, res) => {
+  const itemsData = await itemDB.find();
+  const categorieData = await categorieDB.find();
+
+  res.render("items", { allCategories: categorieData, allItems: itemsData });
+});
+
+// Creating data in categorie collection from user in database
+router.post("/add/categorie", async (req, res) => {
```

Comment:
POST by default means that you are creating something so routes should only contains Noun.
ex: 
router.post("/categories",()=>{})

Reply:
https://learn.microsoft.com/en-us/azure/architecture/best-practices/api-design